src/main/java/it/gov/pagopa/pu/bff/service/organization/OrganizationServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package it.gov.pagopa.pu.bff.service.organization;
+
+import it.gov.pagopa.pu.bff.connector.OrganizationClientImpl;
+import it.gov.pagopa.pu.bff.dto.generated.OrganizationDTO;
+import it.gov.pagopa.pu.p4pa_organization.dto.generated.EntityModelOrganization;
+import it.gov.pagopa.pu.p4paauth.model.generated.UserInfo;
+import it.gov.pagopa.pu.p4paauth.model.generated.UserOrganizationRoles;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class OrganizationServiceImpl implements OrganizationService {
+
+  private final OrganizationClientImpl organizationClient;
```

Comment:
we should always depend on interfaces not implementation
```suggestion
  private final OrganizationClient organizationClient;
```

Reply:
in the other serviceImpl is used the implementation not the interface

Reply:
ok, let's fix it ;)

Reply:
this has not been resolved